components/breadcrumbs/breadcrumb.js

```diff
@@ -0,0 +1,93 @@
+import '../icons/icon.js';
+import '../link/link.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class BreadCrumb extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			compact: {
+				type: Boolean,
+				reflect: true
+			},
+			href: {
+				type: String,
+				reflect: true
+			},
+			target: {
+				type: String,
+				reflect: true
+			},
+			text: {
+				type: String,
+				reflect: true
+			},
+			ariaLabel: {
+				attribute: 'aria-label',
+				type: String,
+				reflect: true
+			}
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				align-items: center;
+				display: inline-flex;
+			}
+
+			div.d2l-breadcrumb-wrapper {
+				align-items: center;
+			}
+
+			:host([compact]) div.d2l-breadcrumb-wrapper {
+				display: flex;
+				flex-direction: row-reverse;
+			}
+
+			d2l-icon {
+				height: 8px;
+				padding-left: 10px;
+				padding-right: 5px;
+				width: 8px;
+			}
+
+			d2l-icon:dir(rtl) {
+				padding-left: 5px;
+				padding-right: 10px;
+			}
+
+			:host([compact]) div.d2l-breadcrumb-wrapper d2l-icon {
+				padding-right: 10px;
+				padding-left: 0;
+			}
+
+			:host([compact]) div.d2l-breadcrumb-wrapper d2l-icon:dir(rtl) {
+				padding-right: 0;
+				padding-left: 10px;
+			}
+		`;
+	}
+
+	constructor() {
+		super();
+		this.href = '#';
+		this.compact = false;
+		this.text = '';
+		this.target = '';
```

Comment:
I would also recommend not initializing and using `ifDefined` for `target` since its optional.

Reply:
Done ✔️